docs/features/colorbutton/README.md

```diff
@@ -72,6 +72,30 @@ CKEditor will then output the color definition as `<font>` elements with `color`
 	<p><font color="#800080">This is my text color.</font><br/>
 	<font style="background-color:#FFFF00;">This is my background color</font></p>
 
+Since CKEditor 4.15.0 it is possible to use `class` instead of `style` to set more advanced styles to text.
+
+You need to set {@linkapi CKEDITOR.config.colorButton_colors custom color names}.
+
+You can set `colorName` to {@linkapi CKEDITOR.config.colorButton_foreStyle CKEDITOR.config.colorButton_foreStyle} or {@linkapi CKEDITOR.config.colorButton_backStyle CKEDITOR.config.colorButton_backStyle} configuration options:
+
+```javascript
+config.colorButton_foreStyle = {
+	element: 'span',
+	attributes: { 'class': '#(colorName)' }
+};
+
+config.colorButton_backStyle = {
+	element: 'span',
+	attributes: { 'class': '#(colorName)' }
+};
+```
+
+CKEditor 4 will then output the color definition as `span` with class:
+
+```html
+<span class="subtext">Text</span>
```

Comment:
This class doesn't really looks like something describing a color, so maybe let's go with something like `text-skyblue` (or any other fancy color name you like)?